src/components/Player.js

```diff
@@ -15,21 +15,18 @@ export default function Player(props) {
   function onError() {
     const internalPlayer = props.playerRef.current?.getInternalPlayer()
     const videoUrl = internalPlayer.getVideoUrl()
-    if (!videoUrl) return
-
     const videoId = getVideoId(videoUrl)
-    const currentVideoIndex = internalPlayer.getPlaylistIndex()
 
+    const currentVideoIndex = internalPlayer.getPlaylistIndex()
     // use JS destructuring syntax to exclude the `description` property from the rest of the object, it can be too long
     const { description, ...destructuredVideo } = props.videos[currentVideoIndex]
-    const errorMessage = `BROKEN VIDEO: ${videoId}, videoUrl: ${videoUrl}, video: ${JSON.stringify(destructuredVideo)}`
 
+    const errorMessage = `BROKEN VIDEO: ${videoId}, index: ${currentVideoIndex}, videoUrl: ${videoUrl}, video: ${JSON.stringify(destructuredVideo)}`
     console.log(errorMessage)
     Honeybadger.notify(errorMessage);
```

Comment:
It might be easier to read if we pass in the other details in as context instead of as the error message itself.
The error message could just be: `errorMessage = `BROKEN VIDEO: ${videoId}` and the context should be the rest of the vars imo.
In Ruby we pass in the context as an additional arg to the `notify` method, I'm guessing the JS version is probably similar.

Reply:
That'll make it easier to read the error message and they'll gather the context for you in the honeybadger details.

Reply:
added context